packages/cozy-harvest-lib/src/components/AccountsList/AccountsListItem.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Account } from 'cozy-doctypes'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import Card from 'cozy-ui/transpiled/react/Card'
+import { Caption } from 'cozy-ui/transpiled/react/Text'
+import { getErrorLocale } from '../../helpers/konnectors'
+
+import withLocales from '../hoc/withLocales'
+import * as triggersModel from '../../helpers/triggers'
+
+export class AccountsListItem extends React.PureComponent {
+  getStatusContent = () => {
+    const { t, trigger, konnector } = this.props
+    const error = triggersModel.getError(trigger)
+    const errorTitle = getErrorLocale(error, konnector, t, 'title')
+
+    if (error) {
+      return (
+        <div className="u-pomegranate u-flex u-flex-justify-center u-flex-items-center">
+          <span className="u-mr-half u-caption u-pomegranate">
+            {errorTitle}
+          </span>
+          <Icon icon="warning" />
+        </div>
+      )
+    } else {
+      return <Icon icon="check-circleless" color="#2BBA40" />
+    }
+  }
+
+  render() {
+    const { account, onClick } = this.props
+
+    const accountName = Account.getAccountName(account)
+    const accountLogin = Account.getAccountLogin(account)
+    const nameAndLoginDiffer = accountName !== accountLogin
+    const statusContent = this.getStatusContent()
```

Comment:
why do we create a `getStatusContent` method and do not write the method in the render directly?

Reply:
It was like that before, but there's no good reason for it.